README.md

```diff
@@ -81,7 +81,7 @@ The status of various storage engines that power the OpenEBS Persistent Volumes
 | Jiva | stable | Best suited for running Replicated Block Storage on nodes that make use of ephemeral storage on the Kubernetes worker nodes |
 | cStor | stable | A preferred option for running on nodes that have Block Devices. Recommended option if Snapshot and Clones are required |
 | Local Volumes | stable | Best suited for Distributed Application that need low latency storage - direct-attached storage from the Kubernetes nodes. |
-| Mayastor | beta | A new storage engine that operates at the efficiency of Local Storage but also offers storage services like Replication. Development is underway to support Snapshots and Clones. |
+| Mayastor | stable | Container native persistent storage solution for Kubernetes, with near-native NVMe performance and advanced data services. |
```

Comment:
Should it be Container-native with a hyphen?

Reply:
Yes, good catch.